addons/overpressure/functions/fnc_overpressureDamage.sqf

```diff
@@ -37,7 +37,7 @@ TRACE_3("cache",_overpressureAngle,_overpressureRange,_overpressureDamage);
         private _distance = vectorMagnitude _relativePosition;
         private _angle = acos (_axisDistance / _distance);
 
-        private _line = [_posASL, _targetPositionASL, _firer, _x];
+        private _line = [_posASL, _targetPositionASL, vehicle _firer, _x];
```

Comment:
Line 33 might also have to be changed, to accommodate gunners being able to be injured from overpressure/backblast from other sources. I'm not sure about this though, so hold off on any changes to that just yet.

Reply:
Checking line intersection between the BB origin and a player on a vehicle seems a bit tricky though, it keeps registering an intersection with 2 guys on a quad right behind an SPG-9 🤔